packages/cli/generators/repository/index.js

```diff
@@ -0,0 +1,393 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+const _ = require('lodash');
+const ArtifactGenerator = require('../../lib/artifact-generator');
+const debug = require('../../lib/debug')('repository-generator');
+const inspect = require('util').inspect;
+const path = require('path');
+const chalk = require('chalk');
+const utils = require('../../lib/utils');
+const connectors = require('../datasource/connectors.json');
+const tsquery = require('@phenomnomnominal/tsquery').tsquery;
```

Comment:
I am afraid `tsquery` is not properly licensed as open-source, which means we cannot add it to our dependencies.

I opened an issue in the project repository - see https://github.com/phenomnomnominal/tsquery/issues/25, hopefully the project maintainers will fix it soon.

Reply:
Nice, they added the License already 👍